src/main/java/evelyn/command/Parser.java

```diff
@@ -0,0 +1,76 @@
+package evelyn.command;
+
+import evelyn.exception.InvalidInputException;
+import evelyn.exception.NoInputException;
+import evelyn.task.Deadline;
+import evelyn.task.Event;
+import evelyn.task.Todo;
+
+import java.time.DateTimeException;
+import java.util.Objects;
+
+/**
+ * Houses all the logic for parsing words that have been inputted
+ */
+public class Parser {
+    private TaskList lst;
```

Comment:
```suggestion
    private TaskList list;
```

could update as such? according to code quality naming.